### PR TITLE
feat: add captcha key config

### PR DESCRIPTION
**Pull request by luizstacio**
When using the faucet on projects locally, we should not display the captcha, and we should also remove the captcha key from the source code.

On this PR I create a new env config for `CAPTCHA_KEY` and add a optional render for the CAPTCHA based on the presence of the CAPTCHA_KEY.

### CODE REVIEW

**Comment by xgreenx** [comment]
What is the difference between `CAPTCHA_SECRET` and `CAPTCHA_KEY`?

**Comment by luizstacio** [comment]
> What is the difference between `CAPTCHA_SECRET` and `CAPTCHA_KEY`?

The `CAPTCHA_SECRET`  is used by the API to verify if the captcha was correctly generated.
The `CAPTCHA_KEY` is a public key used to generate the captcha to be latter validated on the API.

File: deployment/scripts/.env [hunk 1]
```diff
@@ -7,6 +7,7 @@ fuel_faucet_image_repository="ghcr.io/fuellabs/faucet"
 fuel_faucet_image_tag="latest"
 fuel_faucet_human_logging="false"
 fuel_faucet_wallet_secret_key="random"
+fuel_faucet_captcha_key="fuelrocks"
```
Comment (xgreenx) [inline deployment/scripts/.env:10]:
```suggestion
fuel_faucet_captcha_key="6Ld3cEwfAAAAAMd4QTs7aO85LyKGdgj0bFsdBfre"
```

**Review by xgreenx — APPROVED** [review]
Thank you!=) Did you check that it works as expected with proper `fuel_faucet_captcha_key`?

**Pull request merged**